test/charts/trends/trends-chart.spec.js

```diff
@@ -38,11 +38,9 @@ describe('Directive: pfTrendsChart', function() {
   });
 
   var compileChart = function (markup, scope) {
-    element = $compile(angular.element(markup))(scope);
-    scope.$apply();
-    isolateScope = element.isolateScope();
-
-    return element;
+    var el = $compile(markup)(scope);
+    scope.$digest();
+    return el;
```

Comment:
Why not return angular.element(el) here so that you don't have to do this each time in the calling functions?